vagrant/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -4,7 +4,29 @@
     name: samba
     state: present
 
+- name: Prepare CephFS VFS support
+  when: config.be.variant == 'vfs'
+  block:
+    - name: Install CephFS VFS module
+      yum:
+        name: samba-vfs-cephfs
+        state: present
+
+    - name: Set selinux in permissive mode
+      selinux:
+        policy: targeted
+        state: permissive
+
+    - name: Skip check of share path
+      lineinfile:
+        path: /etc/ctdb/script.options
+        create: true
+        search_string: "CTDB_SAMBA_SKIP_SHARE_CHECK"
+        line: "CTDB_SAMBA_SKIP_SHARE_CHECK=yes"
+        state: present
```

Comment:
Good catch.

For the default VFS GlusterFS mode we always had `/` and it was not a problem. But with CephFS subvolumes share path can no longer be just `/`. Thanks for investigating the issue.

man [ctdb-script.options(5)](https://ctdb.samba.org/manpages/ctdb-script.options.5.html) also lists an option to have such script-specific options to be defined in a separate file i.e, _50.samba.options_ in this case.

Reply:
I've updated the PR to use the script specific options, but the path of that file can be considered stable ?

Reply:
What's the correct path to add the _.options_ file ?

  - /etc/ctdb/events/legacy/
  - /usr/share/ctdb/events/legacy/

Reply:
> What's the correct path to add the _.options_ file ?
> 
> - /etc/ctdb/events/legacy/

This should be it. As the man page says:
...
_options for SCRIPT are placed in a file alongside the script, with a ".script" suffix added_
..

Those legacy events scripts we enable using [ctdb.setup](https://github.com/samba-in-kubernetes/sit-environment/blob/main/vagrant/ansible/roles/ctdb.setup/tasks/main.yml) role appear as symlinks under /etc/ctdb/events/legacy/.

> - /usr/share/ctdb/events/legacy/

 All available event scripts are located here.

Reply:
Done. Thanks